Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,84 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            
+            char []equal = {'='};
+            string []arr=equation.Split(equal);
+            string right=arr[1];
+            char []multi = {'*'};
+            int a=0,b=0,c=0;
+            string []left=arr[0].Split(multi);
+            string l1=left[0];
+            string l2=left[1];
+            for(int i=0;i<l1.Length;i++){
+                if(l1[i]=='?') a=1;
+            }
+            for(int i=0;i<l2.Length;i++){
+                if(l2[i]=='?') b=1;
+            }
+            for(int i=0;i<right.Length;i++){
+                if(right[i]=='?') c=1;
+            }
+            //Console.WriteLine("a= {0}",l1 );
+            if(a==1){
+                int num1=Int32.Parse(l2);

```

Comment:
Try to reduce the number of if/else, as it makes it difficult to read your code.